cmd/cnbBuild.go

```diff
@@ -353,6 +354,13 @@ func addProjectDescriptorTelemetryData(data *cnbBuildTelemetryData, descriptor p
 }
 
 func callCnbBuild(config *cnbBuildOptions, telemetryData *telemetry.CustomData, utils cnbutils.BuildUtils, commonPipelineEnvironment *cnbBuildCommonPipelineEnvironment, httpClient piperhttp.Sender) error {
+	tempdir, err := ioutil.TempDir("", "cnbBuild-")
```

Comment:
Should we make e.g. the image name a part of the temporary folder name to make this easily identifiable for potential future problem analysis?

Just a thought, I am not sure if this helps or if this is needed for identification.

Reply:
Since temp dir added to the `buildEnvVars`, it will be visible in the logs, so it's possible to correlate without explicit image name being part of the dir